### DISCUSSION TITLE
什么时候能有网页端的“合集和列表”

### DISCUSSION THREAD

**Discussion by wsy461** [ywmoyue/biliuwp-lite#438] [feature]
### 问题描述

现在up放在合集里的视频在个人主页找不到，比如主播的直播回放都是在单独的合集里，主页看不到

### 描述解决方案

up的个人主页可以看到这个功能

### 代码贡献

否

**Comment by GD-Slime** [comment]
你能推荐一些有很多合集的up主吗？我这里正在做测试。

**Reply by minortex** [reply]
学过石油的语文老师，有5个

**Reply by wsy461** [reply]
露米Lumi_Official，有7个

**Reply by wsy461** [reply]
真白花音，9个。vup里比较好找合集多的。

**Reply by wsy461** [reply]
小柔Channel，14个

**Comment by ywmoyue** [comment]
@wsy461 4.6.24已提供该功能，麻烦验证下
